libhb/encavcodec.c

```diff
@@ -68,13 +68,30 @@ static const char * const vpx_preset_names[] =
     "veryfast", "faster", "fast", "medium", "slow", "slower", "veryslow", NULL
 };
 
+static const char * const h26x_nvenc_preset_names[] =
+{
+    "losslesshp", "lossless", "llhp", "llhq", "ll", "bd", "hq", "hp", "fast", "medium", "slow", "default", NULL
```

Comment:
Does llhq have an alias?

Reply:
It's low latency rather than an alias of hx

Reply:
I was thinking "lossless hq" sounded pretty ridiculous. :smile_cat: